tests/python/relay/test_op_level2.py

```diff
@@ -532,6 +532,72 @@ def test_pool2d():
     _test_global_pool2d(relay.nn.global_avg_pool2d, np.mean)
 
 
+def test_pool3d():
+
+    def _test_pool3d_baseline(np_data, in_shape, kernal, strides, padding,
```

Comment:
move this function to topi/python/topi/testing and rename it to pool3d_python
Also use that function to replace the hard coded one in the topi test https://github.com/apache/incubator-tvm/blob/master/topi/tests/python/test_topi_pooling.py#L298-L316

Reply:
comments were incorporated, pls take another look